src/App.jsx

```diff
@@ -1,98 +1,13 @@
 /* eslint-disable max-len */
 
 import './App.scss';
-// import moviesFromServer from './api/movies.json';
+import moviesFromServer from './api/movies.json';
+import { MovieList } from './components/MovieList';
```

Comment:
The correct import statement for a non-default export is without curly braces. Use 'import MovieList from './components/MovieList';' instead.